README.md

```diff
@@ -58,6 +58,7 @@ These are args currently supported:
 - `rbd <args>` : Call a 'rbd' CLI command with arbitrary args
 
 - `mons` : Print mon endpoints
+  - `reset-quorum <mon-name>` : Reset the mon quorum to a single mon since quorum was lost with the other mons
```

Comment:
```suggestion
  - `restore-quorum <mon-name>` : Reset the mon quorum to a single mon since quorum was lost with the other mons
```
how about `restore` instead of `reset`

Reply:
I'm going back and forth between them. When I started I actually called it restore, but then reset seemed more accurate.

Reply:
hm, restore seems better to me but yeah let's have a small discussion and get  others feedback 😅